go.mod

```diff
@@ -0,0 +1,15 @@
+module github.com/go-vela/vela-npm
+
+go 1.14
+
+require (
+	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/cpuguy83/go-md2man/v2 v2.0.1 // indirect
+	github.com/go-vela/types v0.11.0
+	github.com/golang/mock v1.6.0
+	github.com/sirupsen/logrus v1.8.1
+	github.com/spf13/afero v1.8.0
+	github.com/urfave/cli/v2 v2.3.0
+	golang.org/x/sys v0.0.0-20220111092808-5a964db01320 // indirect
+	golang.org/x/text v0.3.7 // indirect
```

Comment:
there are a few dependencies out of date here including go itself. slightly leaning towards taking care of that now vs after this initial merge.

Reply:
@Cappo Is taking some well earned vacation time ATM so I bumped these deps for him.